pex/pex.py

```diff
@@ -444,7 +444,7 @@ def execute_pkg_resources(spec):
     else:
       # setuptools < 11.3
       runner = entry.load(require=False)
-    runner()
+    sys.exit(runner())
```

Comment:
there is the generic case of non-console script entry point execution to consider, esp as used as a library - moving it to `execute_script()` works for me tho.